cspell.config.yaml

```
@@ -1,7 +1,25 @@
 version: "0.2"
-ignorePaths: ["~", "vendor", ".vscode", ".venv"]
+ignorePaths:
+ignorePaths:
```

yamllint (1.35.1)

[error] duplication of key "ignorePaths" in mapping

(key-duplicates)